### PR TITLE
PYIC-7209: update kbv drop out page content

### PR DESCRIPTION
**Pull request by shivanshuit914**
## Proposed changes

- update kbv drop out page content for p1 journey

### What changed

- Updated translation file to show updated content for p1 drop out page.

### Issue tracking

- [PYIC-7209](https://govukverify.atlassian.net/browse/PYIC-7209)

Drop out fail kbV

Screenshot 2024-08-22 at 10 38 14

Dropout thin file

Screenshot 2024-08-22 at 10 37 51

[PYIC-7209]: https://govukverify.atlassian.net/browse/PYIC-7209?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

File: src/locales/en/translation.json [hunk 1]
```diff
@@ -341,9 +335,8 @@
             "UK biometric residence permit (BRP)"
           ]
         },
-        "subHeading3": "What would you like to do?",
+        "subHeading3": "How would you like to prove your identity?",
         "formRadioButtons": {
-          "radioButtonBackToRpText": "Go to the service you need to use and find out if they take other types of ID",
           "radioButtonBackToRpP1Text": "Go to the service you need to use and look for other ways to prove your identity",
```
Comment (blakeyp) [inline src/locales/en/translation.json]: I guess make this `radioButtonBackToRpText` now?

File: src/locales/cy/translation.json [hunk 2]
```diff
@@ -305,16 +305,13 @@
...
       "titleP1": "Mae'n ddrwg gennym, ni allwn gadarnhau eich hunaniaeth y ffordd hyn",
       "titleDropout": "Darganfyddwch ffordd arall i brofi eich hunaniaeth",
-      "titleP1Dropout": "Darganfyddwch ffordd arall i brofi eich hunaniaeth",
       "header": "Mae'n ddrwg gennym, ni allwn gadarnhau eich hunaniaeth y ffordd hyn",
       "headerP1": "Mae'n ddrwg gennym, ni allwn gadarnhau eich hunaniaeth y ffordd hyn",
       "headerDropout": "Darganfyddwch ffordd arall i brofi eich hunaniaeth",
-      "headerP1Dropout": "Darganfyddwch ffordd arall i brofi eich hunaniaeth",
       "content": {
         "paragraph1": "Mae hyn oherwydd nad ydych wedi ateb rhai o'r cwestiynau diogelwch yn gywir.",
         "paragraph1P1": "Mae hyn oherwydd nad ydych wedi ateb rhai o'r cwestiynau diogelwch yn gywir.",
```
Comment (blakeyp) [inline src/locales/cy/translation.json]: still a few P1s in here that need deleting?

File: src/locales/en/translation.json (hunk 1 above)
Comment (shivanshuit914) [reply src/locales/en/translation.json]: Done thanks

File: src/locales/cy/translation.json (hunk 2 above)
Comment (shivanshuit914) [reply src/locales/cy/translation.json]: Done thanks

**Review by blakeyp — APPROVED** [review]

**Pull request merged**